config/protocol.unit_testnet.yml

```diff
@@ -22,6 +22,7 @@ ProtocolConfiguration:
     Basilisk: 6
     Cockatrice: 9
     Domovoi: 12
+    Echidna: 13
```

Comment:
In general I'd prefer at least 1 block gap between subsequent hardforks for unit-tests, because we may need to test some functionality *after* hardfork enabled, but *before* the next hardfork enabled. In this case all related `Domovoi` changes are already tested, so let's keep 13.

Reply:
Yes, I've tried that! But it all fails miserably with `Echidna: 14`. The reason is that we add some transactions in block 14 and they fail with insufficient GAS when Echidna is activated there because their estimate is made pre-Echidna. It can be solved by an empty block, but this will affect other users of the same test chain, so I've left it for someone to untangle when we have more real need in this.